docs/en/observability/troubleshoot-uptime.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[troubleshoot-uptime]]
+= Uptime troubleshooting
+
+== Mapping issues
+
+It is possible for Heartbeat to create an index without the appropriate mappings.
+When this happens, the {uptime-app} will be unable to get the data it requires; in this
+state, the UI cannot function.
+
+To fix this problem, you typically need to remove your Heartbeat indices, and create
+new ones, with the appropriate mappings installed. To achieve this, follow the steps below.
+
+=== Step 1: Delete your Heartbeat indices
+
+["source","sh"]
+----
+curl -u {USERNAME}:{PASSWORD} -X DELETE {INDEX_URL}
+----
+
+=== Step 2: Run the Heartbeat setup command
+
+["source","sh"]
+----
+./heartbeat setup -e 
```

Comment:
Also unsure if it's appropriate to supply this command here, or if we should wholesale link to the page of the Beats book that deals with this command, to avoid this text becoming outdated.

Reply:
Actually, it may make sense to link to https://www.elastic.co/guide/en/beats/heartbeat/current/heartbeat-installation-configuration.html somewhere

Reply:
Link added in https://github.com/elastic/observability-docs/pull/1046/commits/5e439ac02744484eff67ea9612e817cb09c02f71.